### PR TITLE
More flakiness adjustments.

### PR DESCRIPTION
**Pull request by amanda-tarafa**

### CODE REVIEW

File: compute/api/Compute.Samples.Tests/ComputeFixture.cs [hunk 1]
```diff
@@ -54,7 +54,14 @@ public class ComputeFixture : IDisposable, ICollectionFixture<ComputeFixture>
 
         private StorageClient StorageClient { get; } = StorageClient.Create();
 
-        public RetryRobot Assert { get; } = new RetryRobot { RetryWhenExceptions = new Type[] { typeof(XunitException) } };
+        public RetryRobot AssertConcurently { get; } = new RetryRobot
```
Comment (jskeet) [inline compute/api/Compute.Samples.Tests/ComputeFixture.cs:57]: Concurently => Concurrently

File: compute/api/Compute.Samples.Tests/ComputeFixture.cs [hunk 2]
```diff
@@ -54,7 +54,14 @@ public class ComputeFixture : IDisposable, ICollectionFixture<ComputeFixture>
...
+        { 
+            RetryWhenExceptions = new Type[] { typeof(XunitException) },
+            // We wait a little longer than default to let the concurrent test execution
+            // to finish before reattempting.
```
Comment (jskeet) [inline compute/api/Compute.Samples.Tests/ComputeFixture.cs:61]: Nit: remove "to" from the start of this line (the "to" in the previous line is fine).

File: monitoring/api/MonitoringTest/MonitoringTest.cs [hunk 3]
```diff
@@ -12,22 +12,36 @@
...
 namespace GoogleCloudSamples
 {
     public class TestFixture : IDisposable
     {
+        private const string TimeSeriesQuotaError = "One or more TimeSeries could not be written:" +
+            " One or more points were written more frequently than the maximum sampling period configured for the metric.";
+        public static RetryRobot TimeSeriesQuota { get; } = new RetryRobot
+        {
+            ShouldRetry = ex =>
+                (ex is RpcException rpcEx
```
Comment (jskeet) [inline monitoring/api/MonitoringTest/MonitoringTest.cs:29]: Possibly extract the first two expression into method, so you can have

```csharp
ShouldRetry = ex =>
    IsTimeSeriesQuotaError(ex) ||
    ex is AggregateException aggEx && aggEx.Any(IsTimeSeriesQuotaError);
```

I think that would be clearer - assuming I've read the code correctly.

**Review by jskeet — APPROVED** [review]

File: monitoring/api/MonitoringTest/MonitoringTest.cs (hunk 3 above)
Comment (amanda-tarafa) [reply monitoring/api/MonitoringTest/MonitoringTest.cs]: Ah yes, way better, done.

File: compute/api/Compute.Samples.Tests/ComputeFixture.cs (hunk 1 above)
Comment (amanda-tarafa) [reply compute/api/Compute.Samples.Tests/ComputeFixture.cs]: Done

File: compute/api/Compute.Samples.Tests/ComputeFixture.cs (hunk 2 above)
Comment (amanda-tarafa) [reply compute/api/Compute.Samples.Tests/ComputeFixture.cs]: Done

**Review by amanda-tarafa — COMMENTED** [review]
Thanks! All comments addressed.

**Pull request merged**